lib/css/modules/_commentBoxes.scss

```diff
@@ -41,6 +41,6 @@
 	}
 
 	&.res-commentHoverBorder .content .comment:hover {
-		border: 1px solid #99AAEE !important;
+		outline: 1px solid #99AAEE !important;
```

Comment:
Hmmm. This will double-up on the existing gray border instead of just changing its color, and it doesn't play nice with commentRounded.

It might be preferable to use `border-color: #99AAEE !important`, which does (virtually) the same thing without those downsides. If a subreddit removes all of the borders it won't work, but that's still the status quo.